ci/conda_requirements.txt

```diff
@@ -0,0 +1,11 @@
+pip
+nose
+pep8
+flake8
+IPython>4.0.0
```

Comment:
Is this a requirement from the toy example prep? It is not needed in any code that has been pushed so far.

Reply:
yeah It will be needed for the toy example.

Reply:
Can you make sure these are reflected in resources/environment.yml ?

Reply:
do we need them for the env? we should just need them in travis. it will do the testing for us.

Reply:
Is there a reason Travis should need anything our environment does not normally have?

Reply:
nevermind, I see below now.

Reply:
removed entierly.